src/main/java/io/stargate/sgv2/jsonapi/service/cqldriver/executor/TableBasedSchemaCache.java

```diff
@@ -10,12 +10,13 @@
 import io.stargate.sgv2.jsonapi.service.schema.collections.CollectionSchemaObject;
 import io.stargate.sgv2.jsonapi.service.schema.collections.CollectionTableMatcher;
 import java.time.Duration;
+import java.util.Optional;
 
 /** Caches the vector enabled status for the namespace */
 // TODO: what is the vector status of a namespace ? vectors are per collection
```

Comment:
Should remove these TODOs?

Reply:
will leave them until we refactor the schema caching so we think about them when doing it see https://github.com/stargate/data-api/issues/2070